src/omnisharp/server.ts

```diff
@@ -380,20 +381,32 @@ export class OmniSharpServer {
                 });
             }
 
-            // If there's more than one launch target, we start the server if one of the targets
-            // matches the preferred path. Otherwise, we fire the "MultipleLaunchTargets" event,
-            // which is handled in status.ts to display the launch target selector.
-            if (launchTargets.length > 1 && preferredPath) {
+            const defaultLaunchSolution = launchTargets.filter((a) => (a.isDefault === true));
```

Comment:
> const defaultLaunchSolution = launchTargets.filter((a) => (a.isDefault === true)); [](start = 11, length = 83)

I wonder if we can avoid spreading the default launch target selection through 2 different components. 

If we're going to scan this list for the "Default" one anyway, why not just read the option here too?